appveyor.yml

```diff
@@ -59,8 +57,6 @@ test_script:
     - cd c:\php && 7z x php-7.1.3-Win32-VC14-x64.zip -y >nul && copy /Y php.ini-min php.ini
     - cd c:\projects\symfony
     - php phpunit src\Symfony --exclude-group benchmark,intl-data || SET X=!errorlevel!
-    - cd c:\php && 7z x php-5.5.9-nts-Win32-VC11-x86.zip -y >nul && copy /Y php.ini-min php.ini
-    - cd c:\projects\symfony
     - SET SYMFONY_PHPUNIT_SKIPPED_TESTS=phpunit.skipped
     - php phpunit src\Symfony --exclude-group benchmark,intl-data || SET X=!errorlevel!
```

Comment:
this line should be removed, as it was the one running tests on 5.5.9. there is no need to run tests a second time on 7.1.3 without changing the php.ini (skipped tests will stay the same)